src/metabase/driver/druid/query_processor.clj

```diff
@@ -84,7 +84,7 @@
      ::total              (merge defaults {:queryType :timeseries})
      ::grouped-timeseries (merge defaults {:queryType :timeseries})
      ::topN               (merge defaults {:queryType :topN
-                                           :threshold i/absolute-max-results})
+                                           :threshold 1000})
```

Comment:
Wasn't sure myself if I should extract druid-specific variables to the common interface object, but since you also think it is how it should properly done I moved it there.

Could you verify that is is OK now?